terraform/setup/appmesh.tf

```diff
@@ -0,0 +1,99 @@
+resource "aws_iam_policy" "appmesh_k8s_iam_policy" {
```

Comment:
would be possible move it to the setup folder? and have a parameter to control whether these policies are needed? so that we keep setup material in one place.

Reply:
I think it's already is setup folder, isn't it?